index.html

```diff
@@ -302,5 +305,6 @@ <h2>
         </ul>
         <hr>
     </footer>
+    <script src="/script.js"></script>
```

Comment:
Hello @leopiresgoss l , you have indeed done a great job with the mobile menu feature! However, I have noticed that when I click on the menu icon, the menu options do not load.

- [ ] Kindly fix the above issue. I may be happening because the `script.js` file does not load. It maybe because of the forward slash added on the url 👉🏻 `<script src="/script.js"></script>`. Try removing it and reference it as below 👇🏻 

```
    <script src="script.js"></script>
```